greenpithumb/greenpithumb.py

```diff
@@ -72,6 +74,15 @@ def pump_if_needed(self, _):
     ]
 
 
+def make_camera_poller(photo_interval, image_path, record_queue):
+    local_clock = clock.LocalClock()
+    poller_factory = poller.SensorPollerFactory(local_clock, photo_interval,
+                                                record_queue)
```

Comment:
Yeah, I agree but I couldn't think of anything either. My initial approach was to add a photo_interval parameter to the main sensor factory but that felt klunkier.